test/src/main/scala/scala/test/MoreScalaLibResources.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright 2016 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scala.test
+
+object MoreScalaLibResources {
+  def getGreetings() = get("more hellos")
```

Comment:
how does this actually run? I thought you use `more-hellos` above?

Also, can we add a test that actually runs the code, and not just tests compilation?